### ISSUE TITLE
Lite interface?

### ISSUE THREAD

**Issue by vebaev** [yuliskov/SmartTubeNext#781]
Can we have lite UI which is like original YT?

**Comment by vebaev** [comment]
Also can you hide player buttons? Most of the people use like, dislike and channel, others can be hidden under another button, now is quite crowded UI. And channel button will be great to have the tubmnail.

**Comment by costinmx** [comment]
Frequently I use buttons of Playback mode, PIP, HQ, Playlist and CC. Speed is also useful on news. I rarely use like, dislike and channel... It's a TV app...
IMHO current player buttons for every important use are one of the main improvments of ST Next, over original YT interface. 
You press up, and you have direct access to uper row functions.
You press down, and you have direct access to lower row functions.
You press OK (center) and you have the seekbar control.
Buttons also permanently shows the status of different functions.   I does not seem obtrusive on TV screen.
Fast and direct access. You don't have to press more combinations. This is as good as it gets. Without this - will look just as poor as YT interface.

**Comment by vebaev** [comment]
For me the original simple interface is better.

**Comment by costinmx** [comment]
I can't argue with that. Agree for the "Option to hide", if possible.
But if you don't need all these improments, or most of them, you also have the option of using Smart Youtube TV (also by Yuliskov) which is basically like the Youtube player, no extra features.

**Comment by vebaev** [comment]
Yes but for some reason in smart youtube TV most of the 4k videos appear as 720 or any random low res, probably missing codecs?
